mache/spack/chrysalis_gnu_openmpi.yaml

```
@@ -86,65 +86,65 @@ spack:
       buildable: false
     gcc:
       externals:
-      - spec: gcc@9.2.0
-        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-9.3.0/gcc-9.2.0-ugetvbp
+      - spec: gcc@11.2.0
+        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-9.3.0/gcc-11.2.0-bgddrif
         modules:
-        - gcc/9.2.0-ugetvbp
+        - gcc/9.2.0-bgddrif
```

The version number needs to be changed, too:
```suggestion
        - gcc/11.2.0-bgddrif
```